### PR TITLE
feature logo library component

### PR DESCRIPTION
**Pull request by rajeevkumar-debug**
feature logo library component

### CODE REVIEW

File: projects/antra-ui/src/lib/components/logo/logo.component.spec.ts [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+  it('should create', () => {
+    expect(component).toBeTruthy();
+  });
+
+  it('should show a proper logo with different inputs', () => {
+    // inputs one
+    component.svgPath = '/assets/svg/logo.svg';
+    component.logoText = 'BRS';
+    fixture.detectChanges();
+    const imgElement = fixture.nativeElement.querySelector('img');
```
Comment (Patrick-FRE) [inline projects/antra-ui/src/lib/components/logo/logo.component.spec.ts]: `const imgElement = fixture.nativeElement.querySelector('.logo__svg'); `
.logo__svg is a better selector here.

File: projects/antra-ui/src/lib/components/logo/logo.component.spec.ts [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+    const spanElement = fixture.nativeElement.querySelector('span');
```
Comment (Patrick-FRE) [inline projects/antra-ui/src/lib/components/logo/logo.component.spec.ts]: const textElement = fixture.nativeElement.querySelector('.logo__text');
.logo__text is a better selector here.

File: projects/antra-ui/src/lib/components/logo/logo.component.spec.ts (hunk 1 above)
Comment (rajeevkumar-debug) [reply projects/antra-ui/src/lib/components/logo/logo.component.spec.ts]: done changes

File: projects/antra-ui/src/lib/components/logo/logo.component.spec.ts (hunk 2 above)
Comment (rajeevkumar-debug) [reply projects/antra-ui/src/lib/components/logo/logo.component.spec.ts]: done changes

**Pull request merged**